src/fontra/backends/designspace.py

```diff
@@ -1082,6 +1083,26 @@ def packAxisLabels(valueLabels):
     ]
 
 
+def getPostscriptBlueValues(fontInfo):
+    blueValues = getattr(fontInfo, "postscriptBlueValues", [])
+    otherBluesValue = getattr(fontInfo, "postscriptOtherBlues", [])
+    values = blueValues + otherBluesValue
+    return sorted(values)
+
+
+def getZone(value, blueValues):
+    if value is None:
+        return None
+    for i in range(0, len(blueValues), 2):
+        blueValue = blueValues[i]
+        nextBlueValue = blueValues[i + 1]
+        if value in (blueValue, nextBlueValue):
```

Comment:
It would be clearer to write this:
```python
        if value == blueValue:
            return nextBlueValue - blueValue
        elif value == nextBlueValue:
            return blueValue - nextBlueValue
```

Reply:
thanks. done.